packages/dynamite/dynamite/lib/src/builder/generate_schemas.dart

```diff
@@ -21,11 +21,15 @@ Iterable<Spec> generateSchemas(
 
       // TypeDefs should only be generated for top level schemas.
       if (result is TypeResultBase || result.isTypeDef) {
-        yield TypeDef(
-          (b) => b
+        yield TypeDef((b) {
+          if (!(state.buildConfig.analyzerIgnores?.contains('public_member_api_docs') ?? false)) {
+            b.docs.add('// ignore: public_member_api_docs');
+          }
```

Comment:
Is it required to avoid duplicate ignores?

Reply:
yep